docs/docs/docs/01-core/admin/01-ee/02-audit-logs.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Audit Logs
+slug: /settings/audit-logs
+description: Guide for Audit Logs
+tags:
+  - settings
+  - audit-logs
+---
+
+# Audit Logs
+
+## Summary
+
+Audit Logs provide a way to view the history of all user actions in a Strapi application. This includes actions related to entries (including publish actions), media (including its folders), users, login & logout of admin users, components, roles, and permissions.
```

Comment:
Would it make sense to also define which actions trigger a log point somewhere? E.g. I think they are not added automatically for all events, but there is a fixed list of events that is tracked?

Another thought: only admin API actions trigger this, right? Not content API?

Reply:
Or maybe link to the file on GH? :)